ts/IterableInAppClasses.ts

```diff
@@ -174,6 +179,7 @@ class IterableInAppMessage {
     this.inboxMetadata = inboxMetadata
     this.customPayload = customPayload
     this.read = read
+    this.priorityLevel = priorityLevel
```

Comment:
Adding this constructor param will cause compilation error in unit tests in `ts/__tests__/IterableInApp.spec.ts`.

I wonder why the tests are not failing and showing as passed.